extensions/amp-analytics/0.1/vendors/gtag.js

```diff
@@ -22,6 +22,12 @@ export const GTAG_CONFIG = /** @type {!JsonObject} */ ({
         'dr': 'DOCUMENT_REFERRER',
         'dl': 'SOURCE_URL',
       },
+      'conversion': {
+        'gclsrc': 'QUERY_PARAM(gclsrc)',
+        'hasGcl': 'IF(QUERY_PARAM(gclid, 1, 0)',
```

Comment:
This seems to be missing a closing paren `)`. Same below.

Reply:
also `$IF`

Reply:
wow, my conflict resolution miserably failed